Plugins/Public/npc_control/Main.cpp

```diff
@@ -828,7 +845,7 @@ bool ExecuteCommandString_Callback(CCmds* cmds, const wstring &wscCmd)
 	else if (IS_CMD("aifollow"))
 	{
 		returncode = SKIPPLUGINS_NOFUNCTIONCALL;
-		AdminCmd_AIFollow(cmds);
+		AdminCmd_AIFollow(cmds, cmds->ArgStr(1));
```

Comment:
what about ArgCharname? that would mean you'd do `.aifollow!` to get yourself instead of just `.aifollow` but it reuses all the existing code

Reply:
Thanks @dsyalex, I have swapped this over and confirmed it works. Would you prefer I still keep the check for no parameter and set to admin so `.aifollow` works as well as `.aifollow!` or strip this out?